project/src/main/webapp/event-edit.js

```diff
@@ -18,6 +20,4 @@ async function getEventDetails() {
     `${data['date'].month}/${data['date'].dayOfMonth}/${data['date'].year}`;
   document.getElementById('time').value = data['time'];
   document.getElementById('location').value = data['location'];
-  const link = '/event-details.html?eventId=' + eventId;
```

Comment:
add back? (if you take this out, how is the link populated to see event details?)

Reply:
I was kind of confused because this element doesn't exist in the HTML and I was getting null errors from these lines

Reply:
It's been moved to a different place so that only the host can see the edit button